src/sentry/static/sentry/app/utils/withOrganization.tsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 
 import SentryTypes from 'app/sentryTypes';
 import getDisplayName from 'app/utils/getDisplayName';
-import {Organization} from 'app/types';
+import {Organization, LightWeightOrganization} from 'app/types';
 
 type InjectedOrganizationProps = {
-  organization: Organization;
+  organization: Organization | LightWeightOrganization;
```

Comment:
@EvanPurkhiser Can't you just do: `organization: LightWeightOrganization` since `Organization` is a superset of `LightWeightOrganization`?

Reply:
Actually, it seems we are doing a typecast of organization to `Organization` on line 25 so this change seems very problematic...

Reply:
hmmmmmmmm

Reply:
for posterity we discussed this over slack. @scefali just needed some clarification on how the `InjectedOrganizationProps` worked.